src/opnsense/mvc/app/models/OPNsense/Backup/NextcloudSettings.xml

```diff
@@ -9,6 +9,8 @@
         </enabled>
         <url type="TextField">
             <Required>N</Required>
+            <mask>/^https?:\/\/.+[^\/]$/</mask>
```

Comment:
```suggestion
            <mask>/^https?:\/\/.*[^\/]$/</mask>
```

Sorry for backtracking this, your first one is correct, the trailing [] is already a required atom.

Reply:
applied